### DISCUSSION TITLE
Multiple notification sounds

### DISCUSSION THREAD

**Discussion by raisedintheusa** [element-hq/element-meta#1005] [Ideas]
#### What would you like to do?
Assign different notification sounds for each chat, which is already possible in the web client.

#### Why would you like to do it?
Recognize immediately from the notification sound which chat received the new message and better assess the urgency.

#### How would you like to achieve it?
Implement this feature, which is already available in the web client, in the Android client as well.

### Are you willing to provide a PR?

No

**Comment by if-shouldrs** [comment]
Found [this issue vector-im/element-android#169](https://github.com/vector-im/element-android/issues/169) on the last page of all issues.

Looks like it wasn't added to Element because a native Android feature was introduced in Android 8 to let users change per-contact notifications directly from the settings, but clearly no manufacturers cared because the only way to reach that feature seems to be through an option in the app whose notifications you want to customize, designed by the app developers.

**TL;DR:** Would be great to add a redirect to the settings page in the room settings 🙂.

**Comment by Fmstrat** [comment]
@if-shouldrs while this allows you to set alerts for "silent" rooms, etc, it doesn't allow room-by-room. What's worse, is since one-on-one chats are set to silent by default, it resembles a problem like https://github.com/vector-im/element-android/issues/7702 where private chats are treated like groups. So you have to set "Silent" alerts to have a sound and vibrate, rather than actually be silent. This is a pretty big issue to me unless I'm missing something.

Another good route would be to handle this like Signal does, and allow "merging" of rooms to contacts on the device, so the notification setting for that person overrides.

Or make every room it's own notification channel.

**Comment by kittykat** [comment]
Hi, I'm going to move this to a discussion in Element-meta as the issue applies to iOS and Android, and we'd need to consider fixing it in Element X
